src/Tribe/Main.php

```diff
@@ -2642,6 +2642,7 @@ public function dateToTimeStamp( $date, $hour, $minute, $meridian ) {
 		 * @todo remove - unused
 		 */
 		protected function dateHelper( $date ) {
+			_deprecated_function( __FILE__, '3.11', 'date' );
```

Comment:
it's a protected method so only we would be using it.. i searched through the codebase and confirmed we are not when i made that note. all good if you want to keep it and throw the notice though.